iotqatools/iota_measures.py

```diff
@@ -28,7 +28,7 @@
 # imports 3rd party libs
 import requests
 from iotqatools.iot_tools import PqaTools
-
+import urllib2
```

Comment:
It is used to http encode the query param when a measure is sending
for example /iot/d?k=apíkeyñ?i=..... is encoded to
/iot/d?k=ap%C3%ADkey%C3%B1